libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/EdgeToEdgeUtil.kt

```diff
@@ -0,0 +1,36 @@
+package com.salesforce.androidsdk.ui
+
+import android.graphics.drawable.LayerDrawable
+import android.os.Build.VERSION_CODES.UPSIDE_DOWN_CAKE
+import android.view.View
+import androidx.activity.enableEdgeToEdge
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.res.ResourcesCompat
+import androidx.core.view.ViewCompat
+import androidx.core.view.WindowInsetsCompat
+import androidx.core.view.updatePadding
+import com.salesforce.androidsdk.R.drawable.sf__fix_status_bar
+import com.salesforce.androidsdk.R.id.sf__status_bar_background
+
+// TODO: Remove this in 13.0 after rewriting screens in compose.
+internal fun AppCompatActivity.fixEdgeToEdge(view: View) {
+    if (application.applicationInfo.targetSdkVersion > UPSIDE_DOWN_CAKE) {
```

Comment:
I should actually check this _and_ `SDK_INT` so we don't do anything special on pre-Android 15 devices.